### ISSUE TITLE
"Windows cannot access the specified device, path or file" even running in regular user

### ISSUE THREAD

**Issue by szliu-a** [microsoft/terminal#6824]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Microsoft Windows [Version 10.0.18363.900]
Windows Terminal version (if applicable):
Microsoft.WindowsTerminal_1.0.1811.0_8wekyb3d8bbwe.msixbundle

Any other software?
```
This is a work computer inside an Active Directory domain, if that's relevant at all. I am administrator on my local machine though.
# Steps to reproduce

Install Windows Terminal from the Microsoft Store.
Launch the app as regular user in Start Menu.

# Expected behavior

The application starts.

# Actual behavior

The application immediately throws an error: "Windows cannot access the specified device, path, or file. You may not have the appropriate permissions to access the item."

# Extra notes

This is NOT a duplicate of #1538, as that one is about running in administrator, but mine can't even start in regular user. I can't use it at all.
The problem is the same if I use the file downloaded from GitHub Release or install from Microsoft Store.

**Comment by WSLUser** [comment]
This is still a dupe but forget which issue number it is. Basically this is a known issue but very little to go on for troubleshooting. If you manage to find something in Event Viewer to help that would be great.

**Comment by WSLUser** [comment]
Note that Feedback Hub might be able to gather the necessary info but of course you typically can't use it in a work environment with AD.

**Comment by biochemputer** [comment]
I have the same problem.

**Comment by zadjii-msft** [comment]
Huh. That's certainly a pretty useless error message, isn't it 😄 

Out of curiosity, can you launch the Terminal as you admin account? Maybe that'll help shed some light on the problem.

**Comment by biochemputer** [comment]
File permission issue, see here: https://answers.microsoft.com/en-us/windows/forum/all/solved-some-apps-from-microsoft-store-has/887aef4c-e912-474f-8ba2-bdd2a29d0631

**Comment by zadjii-msft** [comment]
@szliu-a Does the above link help with your problem?

**Comment by szliu-a** [comment]
@zadjii-msft It's the same error if I try to run in admin account. I will try the above link.

**Comment by mu88** [comment]
@zadjii-msft I'm encountering the same problem after installing Windows Terminal `v1.18.3181` via Chocolatey on my fresh Win10 laptop (Version 22H2, OS build `19045.3803`). I already tried fixing the permissions as described [here](https://answers.microsoft.com/en-us/windows/forum/all/solved-some-apps-from-microsoft-store-has/887aef4c-e912-474f-8ba2-bdd2a29d0631), but unfortunately, that didn't solve the issue. As a temporary workaround, I'm using the portable version, but I'd really appreciate further help to solve this issue properly 🙏🏻
